Cargo.toml

```diff
@@ -1,46 +1,42 @@
-[package]
-# This is named with the rust- prefix to match https://github.com/Layr-Labs/rust-kzg-bn254
-# We will remove the prefix and publish as new crates once we combine all of these repos
-# and move them into the eigenda monorepo.
-name = "rust-eigenda-client"
+[workspace]
+resolver = "3"
+
+members = [
+    "crates/rust-eigenda-client",
+    "crates/rust-eigenda-client-v2",
```

Comment:
can you rename to `rust-eigenda-v2-client` everywhere? Otherwise it looks like its the v2 client for eigenda v1.

Reply:
done